File: C2309GDoican/PHP/04-05-2024/de11/app/Http/Controllers/CalculationController.php
<?php

namespace App\Http\Controllers;

use Illuminate\Http\Request;

class CalculationController extends Controller
{
    public function index()
    {
        //Tính toán và làm các cv cần thiết        
        return view('sum');//trả về cho trình duyệt view tên là sum.blade.php
    }
}

File: C2110I/MVC/30-12-2022/myWebApp/Controllers/CalculationController.cs
using Microsoft.AspNetCore.Mvc;

namespace myWebApp.Controllers
{
    public class CalculationController : Controller
    {
        public IActionResult Index()
        {
            return View();
        }
        public IActionResult Sum() { 
            return View(); //Sum.cshtml
        }
    }
}
